### PR TITLE
Fix for Issue #161 - handlePasswordInput(CrudRequest $request)

### PR DESCRIPTION
**Pull request by nathanphelps**
In UserCrudController change from
protected function handlePasswordInput(CrudRequest $request)
to
protected function handlePasswordInput(UpdateRequest $request)

### CODE REVIEW

**Comment by ShamarKellman** [comment]
I recommend using using Illuminate\Http\Request instead of the UpdateRequest

File: src/app/Http/Controllers/UserCrudController.php [hunk 1]
```diff
@@ -107,11 +105,11 @@ public function setup()
     /**
      * Store a newly created resource in the database.
      *
-     * @param StoreRequest $request - type injection used for validation using Requests
+     * @param Request $request - type injection used for validation using Requests
      *
      * @return \Illuminate\Http\RedirectResponse
      */
-    public function store(StoreRequest $request)
+    public function store(Request $request)
```
Comment (ShamarKellman) [inline src/app/Http/Controllers/UserCrudController.php]: This stays as StoreRequest

File: src/app/Http/Controllers/UserCrudController.php [hunk 2]
```diff
@@ -121,11 +119,11 @@ public function store(StoreRequest $request)
     /**
      * Update the specified resource in the database.
      *
-     * @param UpdateRequest $request - type injection used for validation using Requests
+     * @param Request $request - type injection used for validation using Requests
      *
      * @return \Illuminate\Http\RedirectResponse
      */
-    public function update(UpdateRequest $request)
+    public function update(Request $request)
```
Comment (ShamarKellman) [inline src/app/Http/Controllers/UserCrudController.php]: This stays as UpdateRequest

**Comment by tabacitu** [comment]
Thank you @nathanphelps  and @ShamarKellman ! You're totally right, I don't understand how this DIDN'T error in 3.12.1 - I was probably using the old version of CRUD which still had CrudRequest.

Thank you, merged and tagged. A ```composer update``` should fix it for everybody.

Cheers!

**Pull request merged**